doc/classes/RandomNumberGenerator.xml

```diff
@@ -17,6 +17,18 @@
 		<link title="Random number generation">$DOCS_URL/tutorials/math/random_number_generation.html</link>
 	</tutorials>
 	<methods>
+		<method name="randb">
+			<return type="bool" />
+			<param index="0" name="chance" type="float" default="0.5" />
+			<description>
+				Returns a random boolean with a bias of [param chance] to be [code]true[/code] and [code]false[/code] otherwise.
+				[codeblocks]
+				[gdscript]
+				randb(0.5) # Returns true at the param chance of 0.5
```

Comment:
```suggestion
				randb(0.5) # Returns true at the param chance of 0.5.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).